lib/src/migrators/module.dart

```diff
@@ -438,21 +459,22 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
   /// Declares a variable within this stylesheet, in the current local scope if
   /// it exists, or as a global variable otherwise.
   void _declareVariable(VariableDeclaration node) {
-    if (_localScope == null || node.isGlobal) {
-      var name = _unprefix(node.name);
+    var name = node.name;
+    if (_scope.parent == null || node.isGlobal) {
```

Comment:
Consider making `_scope.parent == null` into a getter, such as `_scope.isGlobal`, to better communicate the semantics of what you're checking.